File: 75. Sort Colors/code.java
class Solution {
    public void sortColors(int[] nums) {
        for(int i=1; i<nums.length; i++){
            int p = i;
            
            while(p>0 && nums[p]<nums[p-1]){
                int t = nums[p];
                nums[p] = nums[p-1];
                nums[p-1] = t;
                p--;
            }
        }

    }
}

File: 560. Subarray Sum Equals K/code.cpp
class Solution {
public:
    int subarraySum(vector<int>& nums, int k) {
      
        int sum = 0;
        int ans=0;
        map<int,int> sums;
        sums[0]=1;
        
        for(int i: nums){
            sum += i;
            if(sums.find(sum-k)!=sums.end()){
                ans += sums[sum-k];
            }
            sums[sum]++;
            
        }
        
        return ans;
    }
};
